apps/js-machine-front/src/app/components/styles/snowflakesMedia.css

```diff
@@ -0,0 +1,31 @@
+@media screen and (min-width: 960px) {
```

Comment:
you can add media by css-in-js, please relocate it inside makeStyles function,
if you need an example, you can find code with 'theme.breakpoints.'